datasets/leidingeninfrastructuur/waternetRioolknopen/v1.0.0.json

```diff
@@ -0,0 +1,96 @@
+{
+  "id": "waternetRioolknopen",
+  "type": "table",
+  "version": "1.0.0",
+  "title": "Waternet Rioolnetwerk Knopen",
+  "description": "Alle rioolknopen welke door Waternet beschikbaar wordt gesteld.",
+  "auth": "OPENBAAR",
+  "status": "beschikbaar",
+  "creator": "Datateam Stedelijke Ontwikkeling en Beheer",
+  "publisher": "Datateam Stedelijke Ontwikkeling en Beheer",
+  "authorizationGrantor": "datateamstedelijkeonwikkelingenbeheer@amsterdam.nl",
+  "theme": [
+    "Waternet Rioolnetwerk"
+  ],
+  "owner": "Gemeente Amsterdam",
+  "schema": {
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "type": "object",
+    "additionalProperties": false,
+    "required": [
+      "schema",
+      "id"
+    ],
+    "mainGeometry": "geometrie",
+    "display": "id",
+    "properties": {
+      "id": {
+        "type": "string",
+        "description": "Unieke id van het object."
+      },
+      "schema": {
+        "$ref": "https://schemas.data.amsterdam.nl/schema@v1.1.1#/definitions/schema"
+      },
+      "status": {
+        "type": "string",
+        "description": "Status van de rioolknoop."
+      },
+      "jaarGelegd": {
+        "type": "number",
+        "description": "Het jaar waarin de rioolknoop is gelegd."
+      },
+      "soort": {
+        "type": "string",
+        "description": "Soort rioolknoop"
+      },
+      "type": {
+        "type": "string",
+        "description": "Type rioolknoop"
+      },
+      "subtype": {
+        "type": "string",
+        "description": "Subtype rioolknoop"
+      },
+      "maaiveldniveau": {
+        "type": "number",
+        "description": "Niveau van het maaiveld ten opzichte van NAP in meters."
+      },
+      "funderingstype": {
+        "type": "string",
+        "description": "Type fundering"
```

Comment:
Is hier een vast lijstje van mogelijkheden voor? Dan zou er nog een `enum` (https://schemas.data.amsterdam.nl/docs/ams-schema-spec.html#dom-veld-enum) voor toegevoegd kunnen worden. Dit geldt wellicht ook voor andere velden in deze nieuwe schemas.